CONTRIBUTING.md

```diff
@@ -28,7 +28,7 @@ Once you've cloned a Turf repository, and have a terminal open and currenty in t
   - enable pnpm as a package manager. Requires Node 16+.  Alternatively just `npm install -g pnpm`.
 - `pnpm install`
   - install dependencies and build modules
-- `npm test`
+- `pnpm test`
```

Comment:
@RobinVdBroeck I was under the impression that by running `corepack enable pnpm` as the Turf Contributing doc suggests, that users then won't need to use `pnpm` directly, they can just use `npm` and it will use the symlink it setup to pnpm.  That's what I found in my use.  In that case, can you revert your doc changes?

Reply:
Actually I see now with your addition of `packageManager` to the package.json, that may just replace the need to run `corepack enable pnpm`.

Reply:
And I see that my assumption about use of `npm` running `pnpm` is wrong.

From the corepack docs:
> It's always possible to run a `npm` command within a project configured to be used with another package manager, since Corepack cannot intercept it.

Reply:
I also tested and you do need to `corepack enable pnpm` in order for `pnpm install` to work (unless you already have it installed globally), and for it to pick up the version to use from package.json.  So CONTRIBUTING docs look correct as-is.  I'll resolve this now.